src/applications/vaos/referral-appointments/EpsAppointmentDetailsPage.jsx

```diff
@@ -105,81 +105,95 @@ export default function EpsAppointmentDetailsPage() {
   return (
     <PageLayout>
       {!!appointmentLoaded && (
-        <div
-          className="vaos-appts__appointment-details--container vads-u-margin-top--4 vads-u-border--2px vads-u-border-color--gray-medium vads-u-padding-x--2p5 vads-u-padding-top--5 vads-u-padding-bottom--3"
-          data-testid="appointment-card"
-        >
-          <div className="vaos-appts__appointment-details--icon">
-            <va-icon
-              icon="calendar_today"
-              aria-hidden="true"
-              data-testid="appointment-icon"
-              size={3}
-            />
+        <>
+          <div className="vaos-hide-for-print mobile:vads-u-margin-bottom--0 mobile-lg:vads-u-margin-bottom--1 medium-screen:vads-u-margin-bottom--2">
+            <nav aria-label="backlink" className="vads-u-padding-y--2 ">
+              <va-link
+                back
+                aria-label="Back link"
+                data-testid="back-link"
+                text="Back to appointments"
+                href="/my-health/appointments"
```

Comment:
I think we should use an onclick here something like leave the href too for accessibility
```
onClick={e => {
          e.preventDefault();
          history.push(link);
        }}
```

Reply:
Okay, wasn't sure I needed that or not. I'll add it in.